src/Android/Xamarin.Android/Samples/GeometryEngine/ClipGeometry/ClipGeometry.cs

```diff
@@ -0,0 +1,258 @@
+
+            // Set the initial visual extent of the map view to the extent of the graphics overlay.
+            await _myMapView.SetViewpointGeometryAsync(visibleExtent);
+        }
+
+        private Geometry GetExtentOfGraphicsOverlay(GraphicsOverlay inputGraphicsOverlay, double expansionFactor, SpatialReference spatialReferenceType)
+        {
+            // Get all of the graphics contained in the graphics overlay.
+            GraphicCollection inputGraphicCollection = inputGraphicsOverlay.Graphics;
+
+            // Create a new envelope builder using the same spatial reference as the graphics.
+            EnvelopeBuilder unionEnvelopeBuilder = new EnvelopeBuilder(spatialReferenceType);
+
+            // Loop through each graphic in the graphic collection.
+            foreach (Graphic oneGraphic in inputGraphicCollection)
+            {
+                // Union the extent of each graphic in the envelope builder.
+                unionEnvelopeBuilder.UnionOf(oneGraphic.Geometry.Extent);
+            }
+
+            // Expand the envelope builder by the expansion factor.
+            unionEnvelopeBuilder.Expand(expansionFactor);
+
+            // Return the unioned extent plus the expansion factor.
+            return unionEnvelopeBuilder.Extent;
+        }
+
+        private void ClipButton_Click(object sender, EventArgs e)
+        {
+            try
+            {
+                // Remove the Colorado graphic from the input geometries graphics overlay collection. That way it will be easier 
+                // to see the clip versions of the of the GeometryEngine.Clip operation. 
+                _inputGeometriesGraphicsOverlay.Graphics.Remove(_coloradoGraphic);
+
+                // Create a simple line symbol for the resulting GeometryEngine.Clip operation geometries - it will be the same 
+                // as the one used for the boundary of Colorado.
+                SimpleLineSymbol clipGeometriesSimpleLineSymbol = new SimpleLineSymbol(
+                    SimpleLineSymbolStyle.Solid, System.Drawing.Color.Blue, 4);
+
+                // Create the color that will be used as the fill for the resulting GeometryEngine.Clip operation geometries. It 
+                // will be a semi-transparent, blue color and it will be the same as the one used for the boundary of Colorado.
+                System.Drawing.Color clipGeometriesFillColor = System.Drawing.Color.FromArgb(34, 0, 0, 255);
+
+                // Create the simple fill symbol for the resulting GeometryEngine.Clip operation geometries - comprised of a 
+                // fill style, fill color and outline. It will be the same as the one used for the boundary of Colorado.
+                SimpleFillSymbol clipGeometriesSimpleFillSymbol = new SimpleFillSymbol(
+                    SimpleFillSymbolStyle.Solid, clipGeometriesFillColor, clipGeometriesSimpleLineSymbol);
+
```

Comment:
We could shave a few lines of code if we just got the symbol from the Colorado graphic. Like this:
`SimpleFillSymbol clipGeometriesSimpleFillSymbol = _coloradoGraphic.Symbol as SimpleFillSymbol;`